app/assets/javascripts/components/articles/course_ores_plot.jsx

```diff
@@ -59,6 +80,10 @@ const CourseOresPlot = createReactClass({
         return (
           <div className="ores-plot">
             <CourseQualityProgressGraph graphid={'vega-graph-ores-plot'} graphWidth={1000} graphHeight={200} articleData={this.state.articleData} />
+            <div>
+              <button className="button ghost stacked right" onClick={this.delete}>Delete Cached Data</button>
+              <button className="button ghost stacked right" style = {{ marginRight: 20 }} onClick={this.refresh}>Refresh Cached Data</button>
```

Comment:
I don't think there's any need to do these separately; the only use case for deleting the data is so that you can see a newer version of it, so I think just the 'refresh' version should be included.